monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
+Para RPC-Pay, los hashes son calculados por el cliente RPC mismo o por un ayudante al cliente (con mayor potencia), y los hashes son enviados al servidor a medida que son calculados. Cada hash enviado cuenta como pago. Un porcentaje minúsculo de los hashes que, cuando se representen como un número y minen un bloque por ser lo suficientemente pequeños, eventualmente tendrán valor al servidor. Estos hashes exitosos ocurren de manera aleatoria dentro de los tantos pagos con los hashes. El servidor, luego, cuenta todos los hashes para crédito, pero solo recibe pago a si mismo cuando recibe y publica un hash exitoso en la red de Monero.
 
 ### _RPC-Pay_
 
-RPC-Pay is a new feature added to _monerod_ and wallets that enables payment for RPC calls using mining hashes. This capability is configured through the command line when starting the _monerod_ software. Representative startup parameters are illustrated below in [Example #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2). Similarly, there are commands with the CLI wallet that enable it to create hashes and pay monerod, and these are also presented in [Example #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2).
+RPC-Pay es una nueva característica añadida al _monerod_ y los monederos que habilitan pagos para las llamadas RPC, utilizando hashes de minería. Esta capacidad está configurada a través de la línea de comando al iniciar el software del _monerod_. Parámetros específicos de inicio se ilustran más abajo en el [Ejemplo #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2). De manera similar, hay comandos con el monedero CLI que lo habilitan para crear hashes y pagarle al monerod. Estos también están representados en el [Ejemplo #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2).
 
-With the current Monero release, there is a _monerod_ command (of the type that can be typed right into the _monerod_ console) related to RPC, rpc_payments. This command displays information on paying clients and their balances in the console window. Note that a node will forget about credit balances after they are untouched for six months.
+Con el lanzamiento actual de Monero, hay un comando en el _monerod_, rpc_payments, relacionado con el RPC (del mismo tipo de comandos que se pueden escribir directamente a la consola del _monerod_). Este comando muestra información de los clientes que pagan y sus balances en la ventana de la consola. Ten en cuenta que el nodo olvidará los balances de los créditos despues de no haber sido tocados por un periodo de seis meses.
```

Comment:
_**después**_ is a correction, and also a little suggestion:

_Ten en cuenta que el nodo olvidará los balances **de créditos después de no haber sido tocado por seis meses**._